watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -257,18 +349,10 @@ def build(self):
             units=pyunits.Pa * pyunits.s,
             doc="Fluid viscosity",
         )
-        # Ion electrical mobility
-        self.electrical_mobility_comp = Param(
-            self.ion_set | self.solute_set,
-            mutable=True,
-            default=5.19e-8,  # default as Na+
-            initialize=self.config.electrical_mobility_data,
-            units=pyunits.meter**2 * pyunits.volt**-1 * pyunits.second**-1,
-            doc="Ion electrical mobility",
-        )
+
         # Ion charge
         self.charge_comp = Param(
-            self.ion_set | self.solute_set,
```

Comment:
@lbibl Maybe I am missing something, but is it sufficient to make charge_comp indexed on ion_set only when using to evaluate electrical properties. We need to check that user does not input ion sets with '0' charge. Can you please point me where you are raising errors when users does it by mistake ?

Reply:
What "mistake" do you mean?

Reply:
@lbibl I think @srikanthallu is referring to the idea of raising an exception in the case that a user tries entering 0 for the charge of an ion. 

If you were to have such an exception being raised, you'd also want to have a test to ensure that the exception actually gets raised when entering charge = 0 for an ion.

Reply:
Test added.